src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingForm.tsx

```diff
@@ -38,60 +40,55 @@ class ReprocessingForm extends React.Component<Props> {
         </Header>
         <Body>
-          <Alert type="warning">
-            {t(
-              'Reprocessing is a preview feature. Please carefully review the limitations and implications!'
-            )}
-          </Alert>
-
           <p>
             {t(
-              'You can choose to reprocess issues to apply new debug files and updated grouping configuration. While reprocessing is in preview, keep the following limitations in mind:'
+              'Reprocessing applies any new debug files or grouping configuration to an Issue. Right now, this is a preview feature. So before you give it a try, you should probably consider these impacts:'
             )}
           </p>
 
-          <ul>
-            <li>
+          <StyledList>
+            <ListItem>
               {tct(
-                'Sentry [strong:creates new events and deletes this issue.] This may temporarily affect event counts in Discover and the Issue Stream.',
+                '[strong:Creates new events and deletes the old issue.] This may temporarily affect event counts in both Discover and the Issue Stream.',
                 {strong: <strong />}
               )}
-            </li>
-            <li>
+            </ListItem>
+            <ListItem>
               {tct(
-                'Depending on the number of events, [strong:reprocessing can take several minutes.] Once started, Sentry opens a view on the Issue Stream that shows the reprocessed issues.',
+                '[strong:Reprocessing may take a while.] See progress as it happens in the Issue Stream.',
                 {strong: <strong />}
               )}
-            </li>
-            <li>
+            </ListItem>
+            <ListItem>
+              {' '}
               {tct(
-                'To reprocess Minidump crash reports, ensure [strong:storing native crash reports is enabled.] Attachment storage is required for this.',
+                '[strong:Enable Native crash reports to reprocess Minidump crash reports.] Note that this requires attachment storage.',
                 {strong: <strong />}
               )}
-            </li>
-            <li>
+            </ListItem>
+            <ListItem>
               {tct(
-                "Reprocessing one or multiple events [strong:counts against your organization's quota]. Rate limits and spike protection do not apply.",
+                '[strong:Reprocessed events count towards your organization’s quota]. Sorry, but rate limits and spike protection don’t apply here.',
                 {strong: <strong />}
               )}
-            </li>
-            <li>
+            </ListItem>
+            <ListItem>
               {tct(
-                'If you have uploaded missing debug files, [strong:please wait at least one hour before attempting to reprocess.]',
+                '[strong:Wait one hour before attempting to reprocess missing debug files.] Something didn’t work? Take a break, get a snack, try again in an hour.',
```

Comment:
I like the playful phrasing here, although I think we can use the text to clarify why this is so important. 

This is a strong limitation in our platform: Sentry takes up to one hour before it starts to apply newly uploaded symbols. If events are reprocessed before that, then they count towards the quota but nothing changes.

Excerpt from our docs:
![image](https://user-images.githubusercontent.com/1433023/96086187-332d8e80-0ec2-11eb-8b4d-0d0c1c737d6f.png)

Reply:
Ah, so you mean they need to wait 1 hour to reprocess an issue _after_ they’ve uploaded new symbols?

Reply:
Correct, yes.

Reply:
Perhaps just removing the cutesy text here is enough then

Reply:
Haha, indeed.